content/en/real_user_monitoring/mobile_and_tv_monitoring/troubleshooting/ios.md

```diff
@@ -84,6 +84,10 @@ private class YourCustomDelegateURLSessionDelegate: NSObject, URLSessionTaskDele
 * implement `URLSessionDataDelegate` and forward:
   * [`urlSession(_:dataTask:didReceive:)`][6]
 
+## "Deobfuscation failed" warning
+
+A warning appears when deobfuscation fails for a stack trace. If the stack trace is not obfuscated to begin with, you can ignore this warning. Otherwise, use the [RUM Debug Symbols page][7] to view all your uploaded dSYMs. See [Investigate Obfuscated Stack Traces with RUM Debug Symbols][8].
```

Comment:
```suggestion
A warning appears when symbolication fails for a stack trace. If the stack trace is not symbolicated to begin with, you can ignore this warning. Otherwise, use the [RUM Debug Symbols page][7] to view all your uploaded dSYMs. See [Investigate Obfuscated Stack Traces with RUM Debug Symbols][8].
```

As this page is specific to iOS, we should use the appropriate wording:

Variant | File name | Action verb
-- | -- | --
Generic | Debug symbols | Obfuscate, Deobfuscate
iOS | dSYM | Symbolicate, Desymbolicate
Android | Mapping files | –
JS, React Native | Source maps | –
Flutter | Symbol files | –